.github/workflows/test_windows.yml

```diff
@@ -17,7 +17,7 @@ defaults:
     shell: bash
 
 env:
-  WIN_PYTHON_VERSION: 3.10.7
+  WIN_PYTHON_VERSION: 3.11.2
```

Comment:
I guess there is not such thing as manywindow !? It's a shame that only one binary version is distributed for Windows.

Reply:
Well, the difference between Windows and Linux is that Python comes with Linux, so you're dependent on the version delivered (more or less). On Windows, you anyway need to install Python and can decide which version (and we deliver anyway a compiled binary).